test/python/WMCore_t/WMSpec_t/Steps_t/Fetchers_t/PileupFetcher_t.py

```diff
@@ -44,7 +44,7 @@ def setUp(self):
         couchServer = CouchServer(os.environ["COUCHURL"])
         self.configDatabase = couchServer.connectDatabase("pileupfetcher_t")
         self.testDir = self.testInit.generateWorkDir()
-        self.rucioAcct = "wmcore_transferor"
+        self.rucioAcct = "wmcore_pileup"
```

Comment:
Kenyi, can you please remove this fix? It will go in in my PR fixing it on the workqueue layer.